vulcanai/tests/models/test_cnn.py

```diff
@@ -217,3 +217,53 @@ def test_forward_pass_class_not_nan_single_value(self,
             data_loader=test_dataloader,
             transform_outputs=False)
         assert np.any(~np.isnan(raw_output))
+
+    def test_early_stopping(self, conv3D_net_class_early_stopping,
+                            conv3D_net_class):
+        """ Test that their final params are different: aka
+        that the early stopping did something.
+        Constant seed resetting due to worker init function of dataloader
+        https://github.com/pytorch/pytorch/issues/7068 """
+
+        vulcanai.set_global_seed(42)
```

Comment:
does it really need to be set this many times?

Reply:
it does when wanting to get determinacy creating several dataloaders, as they're each drawing from random. Open to other suggestions, I don't see one at the moment however.

Reply:
So we get identical data loaders got it. Yeah I’m sure it’s good.